finetune/encoding/input_encoder.py

```diff
@@ -202,6 +202,8 @@ def encode_multi_input(self, Xs, Y=None, max_length=None, pad_token=None):
             encoded=positions, max_length=max_length, special_tokens=-1
         )
 
+        # needed to match up context with chunks if chunk_long_sequences=True
+        starts = [-1, 0] + locations[:-1] + [-1]
```

Comment:
I don't think we can assume that starts are just the same as shifted ends generically.  This only works for GPT2 based models -- for other models we don't necessarily have the whitespace as part of any token.  May not matter but worth considering.

Reply:
So are you saying that this should break with new roberta? since we're going from the gpt2 tokenizer to roberta's tokenize?